components/feature/top-sites/src/test/java/mozilla/components/feature/top/sites/TopSitesUseCasesTest.kt

```diff
@@ -27,6 +27,18 @@ class TopSitesUseCasesTest {
         )
     }
 
+    @Test
+    fun `RenameTopSiteUseCase`() = runBlocking {
+        val topSitesStorage: TopSitesStorage = mock()
+        val topSite: TopSite = mock()
+        val useCases = TopSitesUseCases(topSitesStorage)
+
+        val title = "New title"
+        useCases.renameTopSites(topSite, title)
+
+        verify(topSitesStorage).renameTopSite(topSite, title)
+    }
+
```

Comment:
Can we move this below RemoveTopSiteUseCase to keep the add/remove tests one after another?